src/dom/Value.ts

```diff
@@ -159,9 +159,32 @@ export interface Value {
   /**
    * For the Struct, List, and SExpression types, returns an array containing the container's
    * nested values; otherwise throws an Error.
+   * If there are duplicate fields in a Struct, then it only returns the last value for that field.
    */
   elements(): Value[];
 
+  /**
+   * For the Struct, List, and SExpression types, returns an array containing the container's
```

Comment:
This PR doesn't seem to implement `allElements()` for `Struct` and `List`. Am I missing it?

Reply:
That should be just `Struct` not `SExpression` and `List`. I will change it.